examples/guided.py

```diff
@@ -171,7 +175,16 @@ def ask_user_questions():
 		else:
 			# packages installed by a profile may depend on audio and something may get installed anyways, not much we can do about that.
 			# we will not try to remove packages post-installation to not have audio, as that may cause multiple issues
-			archinstall.arguments['audio'] = 'none'
+			archinstall.arguments['audio'] = None
+   
+   	# Ask what kernel user wants:
+	kernel = input("1. linux\n2. linux-lts\n3. linux-zen\nchoose a kernel of the following: ")
```

Comment:
Use generic_select instead of an input, passing in an array of choices. Make it loop so we can have multiple selected as discussed in #145.